bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -100,6 +112,17 @@ spec:
             cpu: 10m
             memory: 300Mi
         terminationMessagePolicy: FallbackToLogsOnError
+        lifecycle:
+          preStop:
+            exec:
+              command:
+              - /bin/bash
+              - -c
+              - |
+                set -xe
+                echo "$(date -Iseconds) - stopping ovn-northd"
+                OVN_MANAGE_OVSDB=no /usr/share/ovn/scripts/ovn-ctl stop_northd
```

Comment:
if u will rely on SIGTERM trap, then there should be no need for preStop right ? can both try to do the same at the same time when container is going down,  not sure if that will lead to false errors ?

Reply:
I guess we can get rid of the `preStop` hook.

Reply:
@abhat I'll remove preStop and add a note to SIGTERM that if we remove it (eg 4.9+) we should move the functionality to preStop.